minion/server/core.py

```diff
@@ -160,3 +160,14 @@ def process(job, config, keep_data):
 
         if not keep_data:
             shutil.rmtree(local_repo_path)
+
+def stop_server():
```

Comment:
Looks good for stopping the flask/werkzeug server. However, what about the processes in the `multiprocessing.Pool`? 
We have to somehow stop those as well. You might want to have a look at this: https://docs.python.org/3.5/library/multiprocessing.html#multiprocessing.pool.Pool.close